climpred/constants.py

```diff
@@ -1,4 +1,4 @@
-HINDCAST_METRICS = [
+DETERMINISTIC_HINDCAST_METRICS = [
```

Comment:
I know it might seem like overkill, but it would be nice to have comments above some of these constants explaining their scope. There's a lot of them now.